### DISCUSSION TITLE
How to get data related to the boundary path of hatch entity

### DISCUSSION THREAD

**Discussion by luoyu-123** [mozman/ezdxf#457] [Q&A]
I export a DXF file from a CAD software. I want to use the ezdxf library to parse the data of the relevant boundary path of the hatch entity in this file. What should I do.

This is the graph in the DXF file

![屏幕截图_7](https://user-images.githubusercontent.com/59160360/120885981-3d9dc380-c61e-11eb-8cb4-69af6a24598d.png)

**Comment by chibai** [comment]
```python
available_boudPaths = entity.paths.rendering_paths(hatch_style=entity.dxf.hatch_style)
```
try this

**Reply by luoyu-123** [reply]
thanks
